src/main/scala/sbtprotoc/ProtocPlugin.scala

```diff
@@ -271,7 +269,8 @@ object ProtocPlugin extends AutoPlugin {
         .value,
       PB.protocExecutable := {
         import CacheImplicits._
-        import sbt.librarymanagement.LibraryManagementCodec._
+        import sbt.librarymanagement.LibraryManagementCodec.ModuleIDFormat
```

Comment:
avoid importing `sjsonnew.BasicJsonProtocol` twice (line above and here), causing ambiguous implicit resolution